app/assets/javascripts/common/reports.js

```diff
@@ -1297,6 +1337,7 @@ function baseLineGraphConfig() {
     type: "line",
     options: {
       animation: false,
+      clip: false,
```

Comment:
Neat, what does this do?

Reply:
Clip is a new option that crops elements rendered inside the graph/chart at the edge of the graph/chart.
Default `clip: 0` crops 0px from the edges of the graph/chart - this is basically the border of the area.
This results in a cropping of the points that sit along the bottom of the chart.
Setting this to default stops this cropping.
![Screenshot 2023-02-07 at 12 07 41](https://user-images.githubusercontent.com/9541902/217242937-582cc102-da5f-4eb0-b522-edbb0c3d98fb.png)
![Screenshot 2023-02-07 at 12 07 26](https://user-images.githubusercontent.com/9541902/217242954-16204f13-10d6-4f94-91fd-1dd4d4f37eb6.png)